TempFileCleanup.bat

```diff
@@ -124,25 +126,43 @@ if exist %SystemDrive%\$Windows.~WS (
 :: First block handles XP/2k3, second block handles Vista and up
 :: Read 9 characters into the WIN_VER variable. Only versions of Windows older than Vista had "Microsoft" as the first part of their title,
 :: so if we don't find "Microsoft" in the first 9 characters we can safely assume we're not on XP/2k3.
-if /i "%WIN_VER:~0,9%"=="Microsoft" (
+if %WIN_VER_NUM% lss 6.0 (
```

Comment:
Can you update the comment describing this section to explain the new logic?

Reply:
Hi @bknickelbine , can you please comment on the reason for the changed logic in this section?